README.md

```diff
@@ -55,7 +55,7 @@ The NJWDS package also includes pre-compiled files in the `src/` directory. Spec
 
 This builds USWDS styles, builds the Fractal docs, and then deploys them to the `gh-pages` branch. The deployed docs can be found [here](https://newjersey.github.io/njwds).
```

Comment:
[pebble] We should update Step 3 under **Steps to test** (PR cover template) to reference `njwds` instead of `uswds`. 

We should also update other references of `USWDS` to `NJWDS` for consistency/accuracy.